packages/fxa-payments-server/src/lib/config.test.ts

```diff
@@ -155,7 +155,10 @@ const expectedMergedConfig = {
       jenny: '8675309',
     },
   },
-  sentryDsn: '',
+  sentry: {
+    dsn: '',
+    url: '',
```

Comment:
One of [the test failures](https://circleci.com/gh/mozilla/fxa/124114?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) looks like it's coming from hereabouts: This [has a default value in `defaultConfig`](https://github.com/mozilla/fxa/blob/436f929ee8ddc0cddbdf65348d75e32adcaa56a6/packages/fxa-payments-server/src/lib/config.ts#L52), so it's not going to be an empty string here in `expectedMergedConfig `